proteus/mprans/SW2DCV.h

```diff
@@ -54,9 +53,9 @@ namespace proteus
 			    int NNZ, //number on non-zero entries on sparsity pattern
 			    int numDOFs, //number of DOFs
 			    double* lumped_mass_matrix, //lumped mass matrix (as vector)
-			    double* h_lstage, //DOFs of solution at last stage
-			    double* hu_lstage, 
-			    double* hv_lstage, 
+			    double* h_old, //DOFs of solution at last stage
```

Comment:
Are this permanent, or just hacks to do FE?

Reply:
They are permanent. The idea is to program calculateResidual as if a FE step is being done and modify the parameter that is being passed depending on which stage of the SSP method is being considered. The modification is done here:

https://github.com/erdc/proteus/blob/mquezada/fix_minor_bugs_in_SWEs/proteus/mprans/SW2DCV.py#L161